lua/weapons/bobs_gun_base.lua

```diff
@@ -224,6 +224,7 @@ function SWEP:FireAnimation()
 
     -- Ironsights logic
     self.RecoilAmount = self.RecoilBack
+    prevThinkBlowback = CurTime()
```

Comment:
Store it per weapon, having it local like this means each weapon uses the same variable